src/sharpedge/_utils/utility.py

```diff
@@ -16,21 +16,20 @@
 class Utility:
     @staticmethod
     def _input_checker(img_array):
-        # Check the image format: must be a numpy array
+        # Check the image array format: must be a numpy array
         if not isinstance(img_array, np.ndarray):
             raise TypeError("Image format must be a numpy array.")
         # Check input image array shape
         if not (img_array.ndim == 2 or (img_array.ndim == 3 and img_array.shape[2] == 3)):
-            raise ValueError(
-            f"Invalid image shape: {img_array.shape}. "
-            "Expected 2D (grayscale) or 3D with 3 channels (RGB)."
-            )
+            raise ValueError("Image array must be 2D or 3D.")
```

Comment:
do you mind changing the msg into:
```suggestion
            raise ValueError("Image array must be 2D grayscale or 3D RGB.")
```
This is because we have other individual functions that use grayscale and RGB terms and we would want these terms to be used consistently across all functions.

Reply:
Yes! that's a good point.